src/Exceptions/MissingBillingInformationException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\JsonApi\Exceptions;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class MissingBillingInformationException extends AbstractException
+{
+    /**
+     * @param string          $detail
+     * @param \Throwable|null $previous
+     */
+    public function __construct(string $detail, \Throwable $previous = null)
```

Comment:
Maybe it's an idea to accept an array of fields we're missing?

Reply:
I was literally just thinking that.

Reply:
Great minds...